src/test/Components/CreateImageWizardV2/steps/Repositories/Repositories.test.tsx

```diff
@@ -102,28 +94,6 @@ describe('repositories request generated correctly', () => {
     },
   ];
 
-  const expectedNginxRepository: Repository = {
```

Comment:
Same here, as far as I'm aware this might be the only test with module_hotfixes. For the sake of not loosing coverage the MSW fixtures should be updated to accommodate the changes to the code and reflect how real data works.

Could be probably split into a separate PR, but the test should be commented out with a note rather than just removed.

Reply:
Fixed, I mocked searching.